spec/requests/users_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    before(:each) do
+      get '/users'
+    end
+
+    it 'returns http success' do
+      expect(response).to have_http_status(:success)
+    end
+    it 'return 200' do
+      expect(response.status).to eq(200)
+    end
+    it 'renders the index template' do
+      expect(response).to render_template('index')
+    end
+    it 'include the list of users' do
+      puts response.body
```

Comment:
**OPTIONAL** - Please remove this line from your file 🙏 to help make your file more readable and easy to understand